src/components/Button/index.js

```diff
@@ -142,6 +131,10 @@ const Button = styled(
   background-color: ${styles.backgroundColor};
   box-shadow: ${styles.boxShadow};
 
+  & + & {
+    margin-left: var(--f-space--m);
+  }
+
```

Comment:
@ugogo I added this here. It's a change in how the component behaves but I think it makes sense. WDYT?

Reply:
Yep LGTM 👍